tests/wpt/meta/css/cssom/cssom-getPropertyValue-common-checks.html.ini

```diff
@@ -16,3 +16,6 @@
 
   [Can't serialize shorthand set to 'initial' when some longhand is missing]
     expected: FAIL
+
+  [Can't serialize shorthand set to initial value when some longhand is missing]
+    expected: FAIL
```

Comment:
This shouldn't be regressing, but it's not your fault, you will need to rebase after #31299